application/helpers/SurveyRuntimeHelper.php

```diff
@@ -1729,6 +1729,15 @@ private function setSurveySettings($surveyid, $args)
         extract($args);
 
         $this->aSurveyInfo                 = getSurveyInfo($this->iSurveyid, App()->getLanguage());
+        if (isset($args['popuppreview']) && $args['popuppreview']) {
+            $this->aSurveyInfo['showxquestions'] = 'N';
+            $this->aSurveyInfo['shownoanswer'] = 'N';
+            $this->aSurveyInfo['showwelcome'] = 'N';
+            $this->aSurveyInfo['showprogress'] = 'N';
```

Comment:
Not needed in All in one

Reply:
Can you elaborate?

Reply:
If survey is All in one page : then showprogress is set to false (but right : only if done in https://manual.limesurvey.org/AfterFindSurvey )